llvm/test/CodeGen/DirectX/cross.ll

```diff
@@ -0,0 +1,57 @@
+; RUN: opt -S  -dxil-intrinsic-expansion  < %s | FileCheck %s --check-prefix=CHECK

```

Comment:
typicall we do `--check-prefix` if one run is different than the other.  in this case they are the same. My recommendation is to drop one of these runs  and remove the check prefix.

Reply:
My point still stands why do you need two runs?